### PR TITLE
Dtspb 2855 new cw flow

### PR DESCRIPTION
**Pull request by MarcusHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2855
https://tools.hmcts.net/jira/browse/DTSPB-2871
https://tools.hmcts.net/jira/browse/DTSPB-2877
https://tools.hmcts.net/jira/browse/DTSPB-2889
https://tools.hmcts.net/jira/browse/DTSPB-2908
https://tools.hmcts.net/jira/browse/DTSPB-2915

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boGenerateGrantPreviewForRegistrarEscalations", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boStopCaseForRegistrarEscalations", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boFindMatchesForRegistrarEscalations", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
-  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boMarkAsReadyToIssue", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boMarkAsReadyToIssue", "UserRole": "caseworker-probate-issuer", "CRUD": "R"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boSelectForQA", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json:37]: I don't understand why the probate-issuer should not be able to update boMarkAsReadyToIssue - can you explain?

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -540,20 +546,22 @@
...
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "makeCaseDormant", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "reactivateDormantCase", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
-  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boMarkAsReadyForExamination", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
-  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boExamineCase", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boMarkAsReadyForExamination", "UserRole": "caseworker-probate-superuser", "CRUD": "R"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boExamineCase", "UserRole": "caseworker-probate-superuser", "CRUD": "R"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "boAddCommentDocs", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
-  {"CaseTypeID": "GrantOfRepresentation", "CaseEventID": "paymentSuccessCase", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
```
Comment (DelythJustice) [inline ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json:556]: Can you explain what can CRUD the paymentSuccessCase is superuser can't - or is it a change to the flow that means this paymentSuccessCase will not need updating?

File: src/functionalTest/resources/expected-html/caseprogress/case-stopped-html.txt [hunk 3]
```diff
@@ -9,7 +9,7 @@
 <p class="govuk-body-s">The case was stopped on <today/> for one of two reasons:</p>
 <ul class="govuk-list govuk-list--bullet">
 <li>an internal review is needed</li>
-<li>further information from the applicant or solicitor is needed</li>
+<li>further information from the applicant or Probate practitioner is needed</li>
```
Comment (DelythJustice) [inline src/functionalTest/resources/expected-html/caseprogress/case-stopped-html.txt:12]: think this is lower case on the story

File: src/main/java/uk/gov/hmcts/probate/transformer/EvidenceHandledTransformer.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            String evidenceHandled = null;
+
+            if ((SOLICITOR.equals(data.getApplicationType())
+                    && noDocumentsRequiredBusinessRule.isApplicable(data))
+                    || (PERSONAL.equals(data.getApplicationType())
+                    && INTESTACY_NAME.equals(data.getCaseType())
+                    && YES.equals(data.getPrimaryApplicantNotRequiredToSendDocuments()))) {
+                evidenceHandled = NO;
+            }
+            data.setEvidenceHandled(evidenceHandled);
```
Comment (DelythJustice) [inline src/main/java/uk/gov/hmcts/probate/transformer/EvidenceHandledTransformer.java]: Notice this setEvidenceHandled to null when the if conditions are not met (as opposed to not setting it at all). Just pointing out to check it is desired.

**Review by DelythJustice — APPROVED** [review]

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json (hunk 1 above)
Comment (MarcusHMCTS) [reply ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseEvent.json:37]: The following states are obsolete
BOReadyForExamination, BOCaseMatchingExamining, BOExamining 

Then the following events are marked to readonly
Mark as ready for examination[boMarkAsReadyForExamination]
Find matches (Examining)[boFindMatchesForReadyForExamining]
Examine case[boExamineCase]
Mark as ready to issue[boMarkAsReadyToIssue]

**Pull request merged**